.coderabbit.yml

```
@@ -0,0 +1,43 @@
+  java:
+    enabled: true
+    include_patterns:
+      - "src/**/*.java"
+    review_depth: thorough
+    security_focus: high
+    performance_analysis: true
+    best_practices: true
+    
+checks:
+  - security_vulnerabilities
+  - performance_issues  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)